warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -145,6 +146,29 @@ public static FunctionalSet<ValueTuple> isNotNull(Object fieldValue) {
                 return FunctionalSet.singleton(getHitTerm(fieldValue));
             }
         }
+        
+        return FunctionalSet.emptySet();
+    }
+    
+    /**
+     * Returns a {@link FunctionalSet} of hit terms found for {@code fieldValue1}. If {@code fieldValue1} is a singular value tuple, a singleton
```

Comment:
The first sentence may need to be adjusted unless I'm misunderstanding the intended functionality.  Can this method's comment be made to reference the single param method's comments, i.e. "behaves similiarly to isNotNull(Object) except on multiple parameters.  When evaluating multiple parameters, every parameter must meet the criteria or an emptySet will be returned."  I'm not sure that's accurately describing the intention.

Reply:
There have been significant changes from the initial MR, including the unit tests.  Can you please re-review when you have a chance.  Thanks!